schema/deploy/computed_columns/project_revision_project_contacts_status.sql

```diff
@@ -0,0 +1,29 @@
+-- Deploy cif:computed_columns/project_revision_project_contacts_status to pg
+-- requires: util_functions/get_form_status
+
+begin;
+
+create or replace function cif.project_revision_project_contacts_status(cif.project_revision)
+returns text
+as
+$computed_column$
+
+select
+    case
+      when (select exists (select get_form_status from cif.get_form_status($1.id, 'project_contact') where get_form_status = 'Incomplete'))
+        then 'Incomplete'
+      when (select exists (select get_form_status from cif.get_form_status($1.id, 'project_contact') where get_form_status = 'Attention Required'))
+          then 'Attention Required'
+      when (select count(distinct get_form_status) from cif.get_form_status($1.id, 'project_contact')) = 1
+        and (select (select distinct get_form_status from cif.get_form_status($1.id, 'project_contact'))) = 'Complete'
+          then 'Complete'
+      else 'Not Started'
+    end;
```

Comment:
should we use 
```sql
with form_status as (
  select * from cif.get_form_status($1.id, 'project_contact')
)
```
to make this a bit more readable? Not sure if that's improving performance, I assume postgres optimizes this.